entry/auth.h

```diff
@@ -0,0 +1,34 @@
+#include <iostream>
+#include <unistd.h>
+#include <openssl/sha.h>
+#include "crow.h"
+
+
+#define CLIENTIDLEN 4
+#define APIKEYLEN 4
+
+class auth{
```

Comment:
classes should start with capital letters, I'd be surprised if cpplint doesn't complain